grpc-examples/src/main/java/io/stargate/grpcexample/GrpcClientExecuteQuery.java

```diff
@@ -42,6 +42,119 @@ public static void main(String[] args) throws InterruptedException {
     grpcClientExecuteQuery.executeSingleQuery();
     grpcClientExecuteQuery.executeBatchQueries();
     grpcClientExecuteQuery.executeAsyncQueries();
+    grpcClientExecuteQuery.executeStreamingQuery();
+    grpcClientExecuteQuery.executeStreamingBatchQueries();
+  }
+
+  private void executeStreamingBatchQueries() throws InterruptedException {
+    executeStreamingBatchInsertQueries();
+    executeStreamingSelectQuery();
+  }
+
+  private void executeStreamingQuery() throws InterruptedException {
+    executeStreamingInsertQuery();
+    executeStreamingSelectQuery();
+  }
+
+  private void executeStreamingSelectQuery() throws InterruptedException {
+    CountDownLatch responseRetrieved = new CountDownLatch(1);
+    StreamObserver<QueryOuterClass.StreamingResponse> responseStreamObserver =
+        new StreamObserver<QueryOuterClass.StreamingResponse>() {
+          @Override
+          public void onNext(QueryOuterClass.StreamingResponse value) {
+            System.out.println("Select streaming response: " + value);
+            responseRetrieved.countDown();
+          }
+
+          @Override
+          public void onError(Throwable t) {
+            System.out.println("Error: " + t);
+          }
+
+          @Override
+          public void onCompleted() {
+            System.out.println("Select StreamObserver completed");
+          }
+        };
+
+    StreamObserver<QueryOuterClass.Query> queryStreamObserver =
+        asyncStub.executeQueryStream(responseStreamObserver);
+
+    queryStreamObserver.onNext(
+        QueryOuterClass.Query.newBuilder().setCql("SELECT k, v FROM ks.test").build());
+    queryStreamObserver.onCompleted();
+    responseRetrieved.await();
```

Comment:
remove the `responseRetrieved`, it's async code and this is not needed, could confuse people